### ISSUE TITLE
Breaks pi audio

### ISSUE THREAD

**Issue by jg00dman** [pimoroni/speaker-phat#7]
This installer ought to come with a warning! It breaks audio on the pi. I can no longer plug in my headphones. The onboard audio is now missing. In addition the volume control has been removed from the task bar and if I add it again it doesn't work!

Ok, it works but it is not integrated well.

**Comment by Gadgetoid** [comment]
This is necessary to get Speaker pHAT working reliably irrespective of your configuration. That said, it doesn't break onboard audio, it disables it.

Disabling the Volume control is also necessary, since it *only* works with the onboard audio and does some hacky things that will break the pHAT setup. You should still be able to control volume via `alsamixer`. We're unfortunately at odds with the objectives and priorities of the Pixel maintainers over this, although we're looking into ways to configure out way around it.

For the time being, you can edit `/boot/config.txt` and make sure to uncomment `dtparam=audio=on` which should re-enable the onboard audio, however you may need to also modify other configuration to actually use it. Switching between audio devices is a bit of a kerfuffle, which is why we currently disable it outright.

Volume ALSA *tries* to be helpful in this respect, but very naively creates an `~/.asoundrc` that directly sets the DAC as the default output for the current active user. This sounds fine- but that means the software volume control and VU Meter configuration we've painstakingly crafted is just bypassed.

So at the moment you can have:

* Onboard audio enabled and switchable audio via Volume ALSA but no volume control on the Speaker/DAC
* Onboard audio enabled and manually switch outputs or specify on command-line as necessary
* Onboard audio disabled with volume control (via alsamixer) and VU meter

TLDR: It's a complicated can of worms and we're always looking for a better way, but the odds are stacked against us.

Edit: Volume Alsa issue for posterity: https://github.com/raspberrypi/lxpanel/issues/9
